src/content/ja/2019/performance.md

```diff
@@ -0,0 +1,387 @@
+---
+part_number: II
+chapter_number: 7
+title: パフォーマンス
+description: コンテンツの初回ペイント（FCP）、最初のバイトまでの時間（TTFB）、初回入力遅延（FID）を取り扱う2019 Web Almanac パフォーマンスの章。
+authors: [rviscomi]
+reviewers: [JMPerez,obto,sergeychernyshev,zeman]
+discuss: 1762
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2019-11-07T21:46:11.000Z 
```

Comment:
```suggestion
last_updated: 2019-11-17T00:00:00.000Z
```

Reply:
We should probably change the published date too. But can probably hold both off until we decide on a go live date for Japanese edition (whether all chapters or just some) and bulk change them all then?

Btw, while on the subject of the meta data, are we adding a translators line to meta data (and the page!) to give the translator credit and act as a primary contact point for translation issues? Should I raise an issue for this?

Reply:
> Btw, while on the subject of the meta data, are we adding a translators line to meta data (and the page!) to give the translator credit and act as a primary contact point for translation issues? Should I raise an issue for this?

Yes and yes! Good call.

Reply:
https://github.com/HTTPArchive/almanac.httparchive.org/issues/507 raised. Will have a go at fixing this myself but raised the issue so people can comment on design.

@MSakamaki will you add the necessary translator(s) meta data to this PR (while we work separately on actually using that meta data)?

Reply:
Great job @bazzadp!
I will fix the translator and last_updated metadata as per #507 @rviscomi